src/collaborative/ot/srt_specific.ts

```diff
@@ -101,3 +105,92 @@ function addPivotCommandAdaptRange<Cmd extends AddPivotCommand | UpdatePivotComm
 }
 specificRangeTransformRegistry.add("ADD_PIVOT", addPivotCommandAdaptRange);
 specificRangeTransformRegistry.add("UPDATE_PIVOT", addPivotCommandAdaptRange);
+
+function adaptChartRange<Cmd extends CreateChartCommand | UpdateChartCommand>(
```

Comment:
A drawback to have the transform be defined here rather than in the charts is that it cannot be extended if we add a new type of chart outside of o-spreadhseet.

Reply:
yup :o i'll just update `transformDefinition` and where it's called then :)